requirements.txt

```diff
@@ -4,10 +4,13 @@ Flask-API==1.1
 Flask-SQLAlchemy==2.4.1
 psycopg2-binary==2.8.3
 pymysql==0.9.3
+git-pylint-commit-hook
 
 # Testing
 nose==1.3.7
 pinocchio==0.4.2
 factory-boy==2.12.0
 coverage==4.5.4
-pylint>=2.4.1
\ No newline at end of file
+pylint>=2.4.1
+pre-commit==1.18.3
```

Comment:
Do we still need `pre-commit`? I think we can remove this since we are manually setting the data in the `.git/hooks/pre-commit`.

Reply:
You are right I guess it should have be git-pylint-commit-hook in the testing section